scripts/install_protoc.sh

```diff
@@ -1,22 +1,22 @@
-#! /bin/bash
+#!/usr/bin/env bash
 set -euxo pipefail
 
-SCRIPT_PARENT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )
-
-VERSION="24.3"
+OSTYPE=$(uname -s)
 OS="linux"
-if [[ "$OSTYPE" == "darwin"* ]]; then
-  # Mac OSX
-  OS="osx"
+if [[ ${OSTYPE,,} =~ darwin ]]; then
+    OS="osx"
 fi
+
 ARCH="x86_64"
-if [[ "$(uname -m)" == "arm"* ]]; then
-  ARCH="aarch_64"
+if [[ $(uname -m) =~ arm ]]; then
+    ARCH="aarch_64"
 fi
 
-mkdir -p ${SCRIPT_PARENT_DIR}/bin
+PROJECT_ROOT=$(dirname "$(realpath "$0")")/..
```

Comment:
... the comment about absolute path is applying _mandatory_ here instead, for the reasons explained by @Al-Pragliola in https://github.com/kubeflow/model-registry/pull/256